src/Test/L0/Listener/AgentL0.cs

```diff
@@ -171,5 +174,43 @@ private JobCancelMessage CreateJobCancelMessage()
                 }
             }
         }
+
+        public static TheoryData<string[], bool, Times> RunAsServiceTestData = new TheoryData<string[], bool, Times>()
```

Comment:
I've considered using this before, but decided not to since the logs created by TestHostContext would be overwritten. For that reason I've opted to just loop over an array within the using scope of TestHostContext. Either way seems fine, just something to be aware of.